astroid/__init__.py

```diff
@@ -60,7 +60,34 @@
 # pylint: disable=wrong-import-order,wrong-import-position,redefined-builtin
 
 # make all exception classes accessible from astroid package
-from astroid.exceptions import *
+from astroid.exceptions import (
```

Comment:
Instead of importing all exceptions individually here, what do you think about adding `__all__` to `astroid.exceptions`?

Reply:
Make sense, I'm going to do it.